cluster/gce/upgrade.sh

```diff
@@ -439,6 +439,75 @@ function do-node-upgrade() {
   echo "== Finished upgrading nodes to ${KUBE_VERSION}. ==" >&2
 }
 
+CURRENT_COREDNS_VERSION=$(${KUBE_ROOT}/cluster/kubectl.sh -n kube-system get deployment coredns -o=jsonpath='{$.spec.template.spec.containers[:1].image}' | cut -d ":" -f 2)
+
+function update-coredns-config() {
+  # Get the current CoreDNS version
+  local -r coredns_addon_path="/etc/kubernetes/addons/0-dns/coredns"
+  local -r tmpdir=/tmp
+  local -r download_dir=$(mktemp --tmpdir=${tmpdir} -d coredns-migration.XXXXXXXXXX)
```

Comment:
QQ: would the script stops if the `mktemp` command fails?

(Just hoping `rm -rf ${download_dir}` below will not mess up something else :) )

Reply:
I added an `exit` in case the command fails